plugins.xml

```diff
@@ -2640,6 +2640,20 @@
 			<certification type="reviewed"/>
 			<description>This release adds support for OPS 3.3.0. It also adds translation for Brazilian Portuguese</description>
 		</release>
+		<release date="2024-11-16" version="3.3.0.3" md5="bbf81f9c47bc77a9b90881b887e5d00a">
+			<package>https://github.com/ajnyga/funding/releases/download/3.3.0.3/funding.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>~3.3.0.0</version>
```

Comment:
@ajnyga, the `~` version syntax was only added with 3.3.0-11; that's why you see above entries with 3.3.0-0 to 3.3.0-10 listed explicitly. This entry won't work with those versions.

Reply:
ahah, I was wondering what that was about. I will fix these

Reply:
@asmecher this should be fixed now